src/VirtoCommerce.CartModule.Data/Services/ShoppingCartSearchService.cs

```diff
@@ -141,7 +98,7 @@ protected virtual IQueryable<ShoppingCartEntity> BuildQuery(ICartRepository repo
             return query;
         }
 
-        protected virtual IList<SortInfo> BuildSortExpression(ShoppingCartSearchCriteria criteria)
+        protected override IList<SortInfo> BuildSortExpression(ShoppingCartSearchCriteria criteria)
```

Comment:
why not moving this implementation to the base class (if it's not there already)?

Reply:
Seems it's different for every service.